functions-and-scope.js

```diff
@@ -52,14 +85,23 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 // averageGrade(grades) geeft 6.642857142857143
 // averageGrade([6, 4, 5]) geeft xxxx
 // averageGrade([8, 9, 4, 6, 10]) geeft xxxx
+console.log("opdracht 2b: ");
 
+function averageGrade(array) {
+    let totalOfGrades = null;
```

Comment:
Ik zou de default waarde wel 0 laten zijn, omdat dit qua datatype overeenkomt met wat het zou moeten zijn
```suggestion
    let totalOfGrades = 0;
```